lib/support/phone.rb

```diff
@@ -3,6 +3,12 @@ class Phone
 
     attr_accessor :country, :number
 
+    CODE_LENGTH_RANGE_BY_COUNTRY_CODE = {
```

Comment:
one final note thought, we use a class variable below with the `@@` syntax, either do the same here or change the `@@countries` variable to be a constant, just want to maintain style